src/WordPress/PluginActions.php

```diff
@@ -27,6 +28,7 @@ public function __construct(DefaultIntegration $defaultIntegration, APIInterface
         $this->wordpressAPI = $defaultIntegration->getIntegrationAPI();
         $this->dataStore = $defaultIntegration->getDataStore();
         $this->logger = $defaultIntegration->getLogger();
+        $this->integration = $defaultIntegration;
```

Comment:
Why did you add this?  I don't see it being used anywhere?  If its not being used lets remove it.  If it is being used lets talk about it.